src/vim-web/react-viewers/panels/loadingBox.tsx

```diff
@@ -67,3 +71,23 @@ function formatProgress (progress: number, mode? : ProgressMode): string {
     return `${(progress / BYTES_IN_MB).toFixed(2)} MB`
   }
 }
+
+export function UltraSuggestion() {
```

Comment:
Based on our conversation earlier today, this is implemented but unused, correct?